pkgs/development/libraries/audio/vamp/default.nix

```diff
@@ -1,28 +1,30 @@
 # set VAMP_PATH ?
-# plugins availible on sourceforge and http://www.vamp-plugins.org/download.html (various licenses)
+# plugins available on sourceforge and http://www.vamp-plugins.org/download.html (various licenses)
 
-{ stdenv, fetchurl, pkgconfig, libsndfile }:
+{ stdenv, fetchFromGitHub, pkgconfig, libsndfile }:
 
 rec {
 
-  vampSDK = stdenv.mkDerivation {
-    name = "vamp-sdk-2.5";
+  vampSDK = stdenv.mkDerivation rec {
+    name = "vamp-sdk-${version}";
```

Comment:
Eval fails because `version` is undefined here (see ofborg checks).

Reply:
@GrahamcOfBorg eval